dev-lang/bas/bas-2.6.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="An interpreter for the classic dialect of the programming language BASIC"
+HOMEPAGE="http://www.moria.de/~michael/bas/"
+SRC_URI="http://www.moria.de/~michael/bas/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	sys-libs/ncurses
+	virtual/libintl
+"
+DEPEND="${RDEPEND}"
+BDEPEND="sys-devel/gettext"
```

Comment:
Needs **app-alternatives/lex** too.

Reply:
I test packages on a pure testing environment that's stripped to a bare minimum. It's not a requirement for contributors to go through that effort with their testing. But if you look at the Makefile, it calls **flex**.